src/app/tests/TestAclAttribute.cpp

```diff
@@ -46,18 +47,13 @@ namespace {
 using namespace chip;
 using namespace chip::Access;
 
-chip::ClusterId kTestClusterId        = 1;
-chip::ClusterId kTestDeniedClusterId1 = 1000;
-chip::ClusterId kTestDeniedClusterId2 = 3;
-chip::EndpointId kTestEndpointId      = 4;
-
 class TestAccessControlDelegate : public AccessControl::Delegate
 {
 public:
     CHIP_ERROR Check(const SubjectDescriptor & subjectDescriptor, const chip::Access::RequestPath & requestPath,
                      Privilege requestPrivilege) override
     {
-        if (requestPath.cluster == kTestDeniedClusterId2)
+        if (requestPath.cluster == chip::Test::kTestDeniedClusterId2)
```

Comment:
Why the extra `chip::`?

Reply:
because the pigweed namespace is also called `Test`... putting only Test chooses `pw::unit_test::internal::Test`